### PR TITLE
Fixes: Quote carousel 

### PR DESCRIPTION
**Pull request by jzhzhang66**
- Fixed any padding inconsistencies for quote carousel.
- Added string length error checking with ellipses for the about page carousel.

### CODE REVIEW

File: components/about-page/AboutPage.style.js [hunk 1]
```diff
@@ -31,4 +31,23 @@ const SquiggleUnderline = styled.img`
...
+    margin-left: 0.7em;
+  }
+  @media ${min.tablet} {
+    margin-left: 0 0 0 1em;
+  }
+  @media ${min.tabletLg} {
+    margin-left: 0 0 0 1em;
+  }
+  @media ${min.desktop} {
+    margin-left: 0 0 0 6em;
```
Comment (isabellberzsenyi) [inline components/about-page/AboutPage.style.js]: do you mean to do `margin: 0 0 0 xem` or `margin-left: xem`

**Review by isabellberzsenyi — APPROVED** [review]

File: components/about-page/AboutPage.style.js (hunk 1 above)
Comment (jzhzhang66) [reply components/about-page/AboutPage.style.js]: good catch

File: components/story-inquiry-page/StoryInquiryPage.js [hunk 2]
```diff
@@ -81,7 +81,7 @@ export default function StoryInquiryPage({
 
   const quotes = authorTestimonies.map((q) =>
     getString(q.author_testimony).length > 150
```
Comment (jean-zhang) [inline components/story-inquiry-page/StoryInquiryPage.js]: Hmm, seems like a useful utility function?

**Review by jean-zhang — APPROVED** [review]

File: components/quote-carousel/QuoteCarousel.style.js [hunk 3]
```diff
@@ -41,7 +41,6 @@ const ContentWrapper = styled(motion.div)`
 const TextWrapper = styled.div`
```
Comment (jean-zhang) [inline components/quote-carousel/QuoteCarousel.style.js:41]: Screen Shot 2021-05-12 at 12 50 01 PM
did the image rendering get messed up

**Review by jean-zhang — CHANGES_REQUESTED** [review]

File: components/quote-carousel/QuoteCarousel.style.js (hunk 3 above)
Comment (isabellberzsenyi) [reply components/quote-carousel/QuoteCarousel.style.js:41]: fixed in another PR

**Review by jean-zhang — APPROVED** [review]
gorgeous :)))

**Pull request merged**